src/test/java/steps/SuccessfulAccountSteps.java

```diff
@@ -0,0 +1,96 @@
+package steps;
+
+import io.cucumber.java.Before;
+import io.cucumber.java.BeforeAll;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import models.SuccessfulAccountInfo;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import pages.LoginPage;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SuccessfulAccountSteps {
+
+   WebDriver driver = new FirefoxDriver();
+    private LoginPage loginPage = new LoginPage(driver);
+
+    @BeforeAll
+    public static void setup() {
+
+        WebDriverManager.firefoxdriver().setup();
+
+    }
+
+    @Before
+    public void the_user_is_on_dbank_homepage() {
+
+        driver.get("https://dbank-qa.wedevx.co/bank/login");
```

Comment:
How do you think about making this taken from the feature file like we used to do before?